features/support/menu_dishes.feature

```diff
@@ -21,4 +23,4 @@ Scenario: A customer sees a list of dishes divided into categories
 Scenario: A customer sees a list of dishes on the menu page
   Given I visit the menu page
   Then I should see "Burger Special"
-  And show me the page
+  #And show me the page
```

Comment:
Remove the comment here because you will not need it again and incase you need it just type it back in